examples/Examples/Transactions/GetBlockItemStatus/Program.cs

```diff
@@ -0,0 +1,112 @@
+using CommandLine;
+using Concordium.Sdk.Client;
+using Concordium.Sdk.Types;
+
+// ReSharper disable ClassNeverInstantiated.Global
+// ReSharper disable ParameterTypeCanBeEnumerable.Global
+#pragma warning disable CS8618
+
+namespace Concordium.Sdk.Examples.Transactions.GetBlockItemStatus;
+
+internal sealed class GetBlockItemSummaryOptions
+{
+    [Option(HelpText = "Transaction hash to lookup", Required = true)]
+    public string TransactionHash { get; set; }
+
+    [Option(HelpText = "URL representing the endpoint where the gRPC V2 API is served.", Required = true,
+        Default = "http://node.testnet.concordium.com/")]
+    public string Endpoint { get; set; }
+
+    [Option(HelpText = "Port for the gRPC V2 API.", Required = true, Default = 20_000)]
+    public ushort Port { get; set; }
+}
+
+internal static class ExampleHelpers
+{
+    public static T Parse<T>(string[] args)
+    {
+        var parserResult = Parser.Default
+            .ParseArguments<T>(args);
+        if (parserResult.Errors.Any())
+        {
+            Environment.Exit(1);
+        }
+        return parserResult.Value;
+    }
+}
+
+public static class Program
+{
+    /// <summary>
+    /// An example showing how one can query transaction status from a node.
+    /// </summary>
+    /// <param name="args">GetBlockItemSummaryOptions
+    /// Example: --endpoint http://node.testnet.concordium.com/ --transactionhash 143ca4183d0bb204000ad08e0fd5792985c808861b97f3b81cb9016ad39d09d2 --port 20000
+    /// </param>
+    public static async Task Main(string[] args)
+    {
+        var options = ExampleHelpers.Parse<GetBlockItemSummaryOptions>(args);
+        var endpoint = new Uri(options!.Endpoint);
+        var transactionHash = TransactionHash.From(options.TransactionHash);
+        var port = options.Port;
+
+        using var client = new ConcordiumClient(endpoint, port);
+
+        Console.WriteLine("Query node...");
+        var transactionStatus = await client.GetBlockItemStatusAsync(transactionHash);
+
+        // Conditional on the transaction status different actions can be handled.
+        switch (transactionStatus)
```

Comment:
Consider using switch expression for consistency, as it's applied elsewhere in the PR:

In example:
_ = transactionStatus switch
{
    TransactionStatusReceived received => HandleTransactionStatusReceived(received),
    TransactionStatusFinalized finalized => HandleTransactionStatusFinalized(finalized),
    TransactionStatusCommitted committed => HandleTransactionStatusCommitted(committed),
    _ => throw new NotImplementedException(transactionStatus.GetType().ToString())
};

Reply:
I unfortunatly can’t do this since the functions are void. The compilers complains with

`Only assignment, call, increment, decrement, await expression, and new object expressions can be used as a statement`